x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/high_count_network_denies.json

```diff
@@ -0,0 +1,34 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "A machine learning job detected an unusually large spike in network traffic that was denied by network ACLs or firewall rules. Such a burst of denied traffic is usually either 1) a mis-configured application or firewall or 2) suspicious or malicious activity. Unsuccessful attempts at network transit, in order to connect to command-and-control (C2), or engage in data exfiltration, may produce a burst of failed connections. This could also be due to unusually large amounts of reconnaissance or enumeration traffic.  Denial-of-service attacks or traffic floods may also produce such a surge in traffic.",
```

Comment:
As above, for consistency of style with the other security jobs, I'd suggest changing the first part of the description here to 

```
Security: Network - Looks for unusually large spikes in network traffic that was denied by network ACLs or firewall rules.
```